### DISCUSSION TITLE
Integration with 1Password

### DISCUSSION THREAD

**Discussion by JamesMatt** [anonaddy/anonaddy#270] [Feature Requests]
1Password is a well-known and reputable password manager, and I wish AnonAddy could integrate it.

Fastmail is already integrated, but having more options is best!
https://support.1password.com/fastmail/
![image](https://user-images.githubusercontent.com/25457860/154980368-7f5aaea8-d226-4c15-b1a9-cc04490ca194.png)

**Comment by beevk** [comment]
I totally agree. I'm waiting for this integration for so long.

**Comment by AngelosNaoumis** [comment]
I think this is not possible as 1password is not open source?
